README.md

```diff
@@ -11,21 +11,24 @@ All contributions are welcomed :)
 
 Please use `Python3.x` for working with the package.
 
-### 1. Kivy installation
-
-Installation of Kivy is buggy, only working way for the author is given below. Please share other methods if found working with the project.
-
-Installation using sources seems to work fine with version `kivy==1.10.1`
-
-Please follow the following link: [Development Version](https://kivy.org/docs/installation/installation.html#installation-devel)
-
-### 2. Fuzzy installation
-
-`pip install fuzzy`
-
-### 3. CorrectMe installation
-
-Currently no installation option is provided. To use the package `cd` into the root directory and import the packages.
+Creating a virtual environment is recommended as the code is in pre-release. Below steps are shown for conda environment.
+
+```bash
+conda create -n cm_env python=3.6 pip
+source activate cm_env # activate cm_env for windows
+cd CorrectMe # The cloned repository directory
+python -m pip install --user --upgrade setuptools wheel
```

Comment:
I can't figure out why we need to build wheels manually. Couldn't we simply use these commands instead - ```pip install <path_to_cloned_repo>``` or ```python setup.py install```?

Reply:
Yes, you are right! My fault, did not do much digging on `setup.py`. Updated the README, though I was not able to install using git directly. I guess I am having some syntax issues. Do you know any way to do it directly from git?

Reply:
I think this should work ```pip install git+https://github.com/nvs-abhilash/CorrectMe.git```.

Reply:
It doesn't seem to be working for me. Getting this error:
![error_correctme](https://user-images.githubusercontent.com/15072945/49693664-4a511600-fb9f-11e8-8b0a-cac2151b6bd6.png)

This seems to be a issue with PIP: [https://github.com/pypa/pip/issues/5271](https://github.com/pypa/pip/issues/5271) 

Have you been facing similar issue or is it working for you?

Reply:
The git URL is for the `master` branch, which doesn't have `setup.py` yet, hence the error. You have to specify the branch in the URL if its not master. Try this `pip install git+https://github.com/nvs-abhilash/CorrectMe.git@installation`.